one-time-project-setup.md

```diff
@@ -25,7 +25,7 @@ Clone the forked repo. Do _not_ clone this inside of another project folder, bec
 Create a new React app within this project folder. **You must perform this within this front-end project folder**.
 
 ```bash
-$ npx create-react-app .
+$ yarn create react-app .
```

Comment:
There was a note in Learn to use the yarn version instead when the command was introduced: https://learn-2.galvanize.com/cohorts/2330/blocks/1449/content_files/react-components/hello-world.md#installation-woes

We didn't update this instruction with the hope that we can figure out a reliable way to get the npx command to work (it worked in C15). There are a bunch of other places in the curriculum that this would need to be updated for, and it's not clear that we want to change them all right now.

It's not ideal, but we decided a single disclaimer was better for the time being than having a bunch of scattered changes to update again.

I'd recommend leaving as is, and we should add an Asana task to either get to an actual fix, or throw in the towel and really update _everything_ to yarn.

Reply:
Got it! Going to close this PR and cut an Asana ticket instead! Thanks!